weasyprint/css/__init__.py

```diff
@@ -273,18 +273,18 @@ def set_computed_styles(cascaded_styles, computed_styles,
     declaration priority (ie. ``!important``) and selector specificity.
 
     """
-    if parent is None:
-        parent_style = None
-    else:
-        parent_style = computed_styles[parent, None]
-
+    parent_style = computed_styles[parent, None] \
+        if parent is not None else None
+    root_style = computed_styles.get((root, None), {})
```

Comment:
Ah, sorry, it’s more subtle than I first thought. `rem` refers to the initial value for `font-size` on the root element, but it still has its usual definition for *other properties* of the root element.

I’ll leave it to you to figure out how to do this exactly. Unless you’d rather I do it, let me know.